lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -56,9 +56,10 @@ function processForProto(result, opts = {}) {
           audit.scoreDisplayMode = 'notApplicable';
         }
       }
-      // Drop raw values. https://github.com/GoogleChrome/lighthouse/issues/6199
-      if (!keepRawValues && 'rawValue' in audit) {
-        delete audit.rawValue;
+      // Drop numeric values until we decide what to do with the optional type in proto.
+      // https://github.com/GoogleChrome/lighthouse/issues/6199
+      if (!keepRawValues && 'numericValue' in audit) {
```

Comment:
not sure how flexible we are in changing the `keepRawValues` parameter name in the lr entry point, but happy to rename if it's not a big deal

Reply:
we can def change it.

Reply:
> we can def change it.

do you want me to change it now and LR will just have to remember to update on the next roll?

Reply:
well, if we are happy with numericValue now shouldn't we just always include it. remove the option entirely.

Reply:
> well, if we are happy with numericValue now shouldn't we just always include it. remove the option entirely.

removed :)